homeassistant/components/fritz/common.py

```diff
@@ -121,6 +116,11 @@ def devices(self) -> dict[str, Any]:
         """Return devices."""
         return self._devices
 
+    @property
+    def mac(self):
+        """Return device mac address."""
+        return self.unique_id
```

Comment:
I think that makes no sense.

Reply:
I get the mac address to allow entities to be linked to the device:

![image](https://user-images.githubusercontent.com/57354320/116356700-e70dc000-a7fb-11eb-8d98-5c257aaa3e50.png)

Reply:
I agree.